src/pages/CreateProposal/ProposalEditor.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+import { ResizingTextArea, TextInput } from 'components/TextInput'
+import { Trans } from '@lingui/macro'
+
+const ProposalEditorHeader = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${({ theme }) => theme.text2};
+`
+
+const ProposalTitle = memo(styled(TextInput)`
+  margin-top: 10.5px;
+  margin-bottom: 7.5px;
+`)
+
+const _ProposalEditor = ({
+  className,
+  title,
+  body,
+  onTitleInput,
+  onBodyInput,
+}: {
+  className?: string
+  title: string
+  body: string
+  onTitleInput: (title: string) => void
+  onBodyInput: (body: string) => void
+}) => {
+  const bodyPlaceholder = `## Summary
+  
+Lorem ipsum dolor sit amet
+  
+## Methodology
+  
+Lorem ipsum dolor sit amet
+
+## Conclusion
+  
+Lorem ipsum dolor sit amet
+  
+  `
+
+  return (
+    <div className={className}>
+      <ProposalEditorHeader>
+        <Trans>Proposal</Trans>
+      </ProposalEditorHeader>
+      <ProposalTitle value={title} onUserInput={onTitleInput} placeholder="Proposal Title" fontSize="1.5rem" />
```

Comment:
Translate? :D 

You can use the t macro if you need a string. t<backtick>Proposal Title<backtick>